prow/cmd/peribolos/BUILD.bazel

```diff
@@ -49,3 +50,79 @@ go_test(
         "//vendor/k8s.io/apimachinery/pkg/util/sets:go_default_library",
     ],
 )
+
+# Usage:
+# bazel run :e2e-dump -- --github-token-path ~/whatever # --dump=whatever etc
```

Comment:
seems like this comment is for the binary below?

Reply:
Applies to all of them